Jenkinsfile

```diff
@@ -27,6 +27,7 @@ node {
             stage('Bootstrap') {
                 sh 'yarn add --dev jest-junit'
                 sh 'google-chrome --version'
+                sh 'yarn add chromedriver --detect_chromedriver_version'
```

Comment:
It's not for diagnosis. It was added because it comes in handy for future cases, where the chrome version has changed in dockerfile. This line detects the chrome version and adds the corresponding chromedriver version.

Reply:
ok got it